pkg/operator/ceph/csi/spec.go

```diff
@@ -516,6 +514,7 @@ func startDrivers(clientset kubernetes.Interface, rookclientset rookclient.Inter
 			return errors.Wrapf(err, "failed to start rbd provisioner deployment: %+v", rbdProvisionerDeployment)
 		}
 		k8sutil.AddRookVersionLabelToDeployment(rbdProvisionerDeployment)
+		logger.Info("successfully started CSI Ceph RBD driver")
```

Comment:
Same suggestion as below
```suggestion
		logger.Info("successfully started CSI Ceph RBD provisioner")
```